packages/app/src/data-sync/repositories/contact.repository.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+// TODO: fully implementation will be https://appnroll.atlassian.net/browse/CP-1049
+export class ContactRepository {
+  constructor() {}
```

Comment:
The repository should have a model as a dependency

Reply:
will be done in https://appnroll.atlassian.net/browse/CP-1049